docs/development.rst

```diff
@@ -493,48 +581,123 @@ The demographic model should include the following:
+is your first time specifying a model using `msprime`, it's worth taking some time to
+read through the `msprime`
+`documentation and tutorials <https://msprime.readthedocs.io/en/stable/tutorial.html>`_.
+
+
+---------------------
+Write parameter table
+---------------------
+
 The parameters used in the implementation must
 also be listed in a csv file in the ``docs/parameter_tables`` directory. This ensures
-that the documentation for this model displays the parameters. We can check that the
-documentation builds properly after implementation by running ``make`` in the docs
-directory and opening the Catalog page from the ``docs/_build/`` directory.
+that the documentation for this model displays the parameters.
 
-If this is your first time implementing a demographic model in `stdpopsim`, it's a good
-idea to take some time browsing the
-`Catalog <https://stdpopsim.readthedocs.io/en/latest/catalog.html>`_
-and species' demographic models in the
-source code to see how existing models are typically written and documented. If you have
-any questions or confusion about formatting or implementing demographic models, please
-don't hesitate to open an `issue <http://github.com/popgensims/stdpopsim/issues>`_ --
-we're more than happy to answer any questions and help get you up and running.
+Take a look at the csv files currently in ``docs/parameter_tables`` for inspiration.
+The csv file should have the format::
 
------------------------------------
-What models are appropriate to add?
------------------------------------
+    Parameter Type (units), Value, Description
 
-`Stdpopsim` supports any demographic model from the published literature that gives
-enough information to be able to define `msprime` demography objects. At a minimum, that
-includes population sizes and the timing of demographic events. These values need to
-either be given in "physical" units (that is, raw population sizes and time units in
-generations), or be able to be converted to physical units using, e.g., mutation rates
-used in the published study.
 
-Note that it is not necessary that the demographic model is attached to a particular
-species. `Stdpopsim` contains a collection of generic models that are widely used in
-developing and testing inference methods. If there is a generic model that does not
-currently exist in our catalog but would be useful to include, we also welcome those
-contributions. Again, you should provide a citation for a generic models, or it
-should be commonly used .
+We can check that the documentation builds properly after implementation by running
+``make`` in the docs directory and opening the Catalog page from the ``docs/_build/``
+directory. See `Documentation`_ for more details.
+
+
+----------------------
+Test the model locally
+----------------------
+
+Once you have written the demographic model function, you should test the model locally
+with ``stdpopsim``. Follow the development `Installation`_ instructions to install the
```

Comment:
I think it's a bit safer/more stable is we use the explicit references, like ``:ref:`sec_installation`` rather than depending on the header titles matchin.